_includes/releases/v22.1/v22.1.11.md

```diff
@@ -0,0 +1,59 @@
+
+- Added a new [cluster setting](../v22.1/cluster-settings.html) `cloudstorage.azure.concurrent_upload_buffers` to configure the number of concurrent buffers used when uploading files to Azure. [#90449][#90449]
+
+<h3 id="v22-1-11-db-console-changes">DB Console changes</h3>
+
+- Requests to fetch table and database [statistics](../v22.1/cost-based-optimizer.html#table-statistics) now have limited concurrency. This may make loading the [Databases page](../v22.1/ui-databases-page.html) slower, but in return should result in making those pages less disruptive. [#90575][#90575]
+- Updated the filter labels from **App** to **Application Name** and from **Username** to **User Name** on the [SQL Activity page](../v22.1/ui-overview.html#sql-activity). [#91294][#91294]
+- Fixed the filter and label style on the **Transactions** filter label on the [SQL Activity page](../v22.1/ui-overview.html#sql-activity). [#91319][#91319]
+- Fixed the filters in the DB Console so that if the height of the filter is large, it will allow a scroll to reach **Apply**. [#90479][#90479]
+- Added a horizontal scroll to the table on **Explain Plan** tab under [**Statement Details**](../v22.1/ui-statements-page.html). [#91329][#91329]
+- Fixed the filter height on the [Sessions page](../v22.1/ui-sessions-page.html) so that the full dropdown is viewable with scroll. [#91325][#91325]
+
+<h3 id="v22-1-11-bug-fixes">Bug fixes</h3>
+
+- Fixed an extremely rare out-of-bounds crash in the [protected timestamp](../v22.1/architecture/storage-layer.html#protected-timestamps) subsystem. [#90452][#90452]
+- Fixed the calculation of the `pg_attribute.attnum` column for indexes so that the `attnum` is always based on the order the column appears in the index. Also fixed the `pg_attribute` table so that it includes stored columns in secondary indexes. [#90728][#90728]
+- [TTL job](../v22.1/row-level-ttl.html#view-scheduled-ttl-jobs) decoding error messages now correctly contain hex-encoded key bytes instead of hex-encoded key pretty print output. [#90727][#90727]
+- Fixed a bug where CockroachDB clusters running inside of a Docker container on macOS, when mounting a host filesystem into the container, could report the total available capacity calculation of the filesystem incorrectly. [#90868][#90868]
+- Fixed the error `invalid uvarint length of 9` that could occur during TTL jobs. This bug could affect keys with secondary tenant prefixes, which includes the serverless environment. [#90606][#90606]
+- Previously, if a primary key name was a reserved SQL keyword, you could not use [`DROP CONSTRAINT, ADD CONSTRAINT`](../v22.1/drop-constraint.html#drop-and-add-a-primary-key-constraint) to change a primary key without having to move the existing primary key to the secondary index. You would receive a `constraint already exists` error. This is now fixed so that you can use `DROP CONSTRAINT, ADD CONSTRAINT` on a primary key named with a reserved SQL keyword. [#91041][#91041]
```

Comment:
nit: I might drop two bits from this and merge them together, like so:

> Previously, if a primary key name was a reserved SQL keyword, attempting to use the [`DROP CONSTRAINT, ADD CONSTRAINT`](../v22.1/drop-constraint.html#drop-and-add-a-primary-key-constraint) statements to change a primary key would result in a `constraint already exists` error. This is now fixed.  [#91041][#91041]

We drop the:
- `without having to move the existing primary key to the secondary index.` bit, which explains the workaround previously required to accomplish this, and
- `so that you can use DROP CONSTRAINT, ADD CONSTRAINT on a primary key named with a reserved SQL keyword`, which just reiterates the first sentence quite closely.

If you feel that either should remain, definitely go for it! Just an optional comment.